Someone forgot to pack wicd.png icon. We will replace it with existing one.

diff -ruN wicd-1.7.2.4.orig/gtk/gui.py wicd-1.7.2.4/gtk/gui.py
--- wicd-1.7.2.4.orig/gtk/gui.py	2013-03-30 21:47:19.802907553 +0000
+++ wicd-1.7.2.4/gtk/gui.py	2013-03-31 08:13:32.876871673 +0000
@@ -205,8 +205,10 @@
 
         self.status_area.hide_all()
 
-        if os.path.exists(os.path.join(wpath.images, "wicd.png")):
-            self.window.set_icon_from_file(os.path.join(wpath.images, "wicd.png"))
+        if os.path.exists(os.path.join(wpath.images, "../../icons/hicolour/128x128/apps/wicd-gtk.png")):
+            self.window.set_icon_from_file(os.path.join(wpath.images, "../../icons/hicolor/128x128/apps/wicd-gtk.png"))
+        else:
+            print 'icon doesn\'t exist %s' % os.path.join(wpath.images, "../../icons/hicolor/128x128/apps/wicd-gtk.png")
         self.statusID = None
         self.first_dialog_load = True
         self.is_visible = True

